app/src/main/kotlin/com/wire/android/ui/userprofile/teammigration/TeamMigrationViewModel.kt

```diff
@@ -81,4 +88,31 @@ class TeamMigrationViewModel @Inject constructor(
             )
         )
     }
+
+    fun postTeamMigration(onSuccess: () -> Unit) {
```

Comment:
The name is still generic one, team migration can be many things.
Maybe we should be more specific, something like : `migrateFromPersonalToTeamAccount`() ?